tutorials/lxd/use-cuda-within-lxd/use-cuda-within-lxd.md

```diff
@@ -0,0 +1,303 @@
+---
+id: gpu-data-processing-inside-lxd
+summary: Accelerate data processing within LXD containers by enabling direct access to your Nvidia GPU's CUDA engine. 
+categories: lxd
+tags: lxd, cuda, nvidia, gpu, big data
+difficulty: 4
+published: 2017-07-12
+author: Canonical Web Team <webteam@canonical.com>
```

Comment:
Hmm. I don't think we can/should use this for everything, just the generic ones that came from websites. Man up and take responsibility!